src/sass/components/_task-table.scss

```diff
@@ -0,0 +1,7 @@
+// Task Tables
+// ===========
+
+.project-task-status {
+  align-items: center;
+  display: flex;
```

Comment:
This does not vertically center the text, and `display: flex` seems to remove table row borders.

![Screenshot 2019-12-21 15 27 01](https://user-images.githubusercontent.com/552316/71313397-72a45f00-2406-11ea-9ac2-c47290b6d4aa.png)

Reply:
I couldn't test it locally, Im *guessing* there is a div inside the td perhaps? Whatever the structure, we will need the parent container set to display: flex; 

If I get this running locally I can test it out more

Reply:
> If I get this running locally I can test it out more

Yeah, it's worth getting it running for you locally again. Has @wlonk already walked you through blowing up your DB and starting from scratch? It's relatively painless on this project.

Reply:
(Just lemme know if you want/need help on that, @stacyk.)